src/components/cqlLibraryLanding/CqlLibraryLanding.tsx

```diff
@@ -52,6 +52,57 @@ function CqlLibraryLanding() {
   const [selectedCQLLibrary, setSelectedCqlLibrary] =
     useState<CqlLibrary>(null);
 
+  const [numberMyCqlLibraries, setNumberMyCqlLibraries] = useState(0);
+  const [numberAllCqlLibraries, setNumberAllCqlLibraries] = useState(0);
+  useEffect(() => {
+    cqlLibraryServiceApi
+      .fetchCqlLibraries(
+        false,
+        curLimit,
+        curPage,
+        "",
+        abortController.current?.signal
+      )
+      .then((data) => {
+        setNumberAllCqlLibraries(data?.totalElements);
+      })
+      .catch((error) => {
+        if (error.message != "canceled") {
+          setSnackBar({
+            message: "An error occurred while fetching the CQL Library!",
+            open: true,
+            severity: "error",
+          });
+        }
+      })
+      .finally(() => {
+        return setLoading(false);
+      });
+    cqlLibraryServiceApi
+      .fetchCqlLibraries(
+        true,
+        curLimit,
+        curPage,
+        "",
+        abortController.current?.signal
+      )
+      .then((data) => {
+        setNumberMyCqlLibraries(data?.totalElements);
```

Comment:
Add a number (in parentheses) in the My Libraries tab that displays the number of library sets in the tab (that is the number of libraries at the top level of the nested library hierarchy that was implemented in [MAT-8038](https://jira.cms.gov/browse/MAT-8038))
Add a number (in parentheses) in the All Libraries tab that displays the number of library sets in the tab (that is the number of libraries at the top level of the nested library hierarchy that was implemented in [MAT-8038](https://jira.cms.gov/browse/MAT-8038))

According to story, we need to display the count of measure set and not the total count of measures.

Reply:
@RohitKandimalla This will be fixed by new MAT-8038 fix.